spec/plugins/partner_sku_reviews_spec.coffee

```diff
@@ -535,6 +540,13 @@ describe 'PartnerSkuReviews', ->
           it 'displays rating stars', ->
             expect(@subject.querySelector('.sa-star-rating').innerHTML.trim()).to.not.be.empty
 
+          context 'when the review is regarding a purchased product', ->
```

Comment:
Let's also test what happens when the reviews is not associated with a purchased product:

```
context 'when review is associated with a purchased product', ->
  beforeEach ->
    @subject = parent_doc.querySelectorAll('#\\@\\@flavor-product-reviews-extended .sa-reviews-list .sa-review')[0]

  it 'displays the verification mark', ->
    expect(@subject.querySelector('.sa-verification-mark').innerHTML.trim()).to.not.be.empty

context 'when review is not associated with a purchased product', ->
  beforeEach ->
    @subject = parent_doc.querySelectorAll('#\\@\\@flavor-product-reviews-extended .sa-reviews-list .sa-review')[1]

  it 'does not display the verification mark', ->
    expect(@subject.querySelector('.sa-verification-mark')).to.not.exist
```